src/tests/ftest/util/server_utils_params.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, filename, common_yaml):
         #       is set for the running process. If group look up fails or user
         #       is not member, use uid return from user lookup.
         #
-        default_provider = os.environ.get("CRT_PHY_ADDR_STR", "ofi+tcp;ofi_rxm")
+        default_provider = os.environ.get("CRT_PHY_ADDR_STR", "ofi+tcp")
```

Comment:
not part of your change, but this should be also reading out D_PROVIDER .. ugh..

Reply:
yeah agree but nothing has been converted yet (even if you make it sound like it should have been? a grep of CRT_PHY_ADDR_STR shows me tons of files still have it) so I won't be addressing this in this PR. That should be done properly in a separate PR.